packages/manager/src/features/CloudPulse/Utils/FilterConfig.ts

```diff
@@ -3,8 +3,11 @@ import { CloudPulseSelectTypes } from './models';
 import type { CloudPulseServiceTypeFilterMap } from './models';
 
 const TIME_DURATION = 'Time Range';
+export const DBAAS_CAPABILITY = 'Managed Databases';
```

Comment:
why we need this constant any more, if we are adding this capability in only one place